core-services/cluster-profiles/_config.yaml

```diff
@@ -1,6 +1,8 @@
-- profile: example-profile-name
+- profile: aws-opendatahub
```

Comment:
Wouldn't it be nice if we could declare the profile as an array, thus eliminating the need to add multiple configurations for each cluster profile?